client/packages/common/src/intl/locales/en/common.json

```diff
@@ -16,6 +16,7 @@
   "auth.timeout-message": "You have been logged out of your session due to inactivity. Click OK to return to the login screen.",
   "auth.timeout-title": "Session Timed Out",
   "auth.unauthenticated-message": "You are not currently logged in. Click OK to return to the login screen.",
+  "authorisePurchaseOrder": "Authorise Purchase Orders",
```

Comment:
Can this one be removed?

Reply:
Removed! https://github.com/msupply-foundation/open-msupply/pull/8733/commits/bec3a85a5b14c9e04a75841b1334c8ccf478894b